parser-wrapper/interface-impl-v1/src/main/java/org/mule/raml/implv1/model/ActionImpl.java

```diff
@@ -99,6 +99,11 @@ public IResource getResource()
         return map;
     }
 
+    @Override
+    public IParameter getQueryString() {
+        return null;
```

Comment:
Another option could be throwing an UnsupportedOperationException

Reply:
Is not an option because this method gets called when using 1.0 and 0.8, in case of 0.8 it will thrown an exception

Reply:
Ok, but you implemented this way for RAML 0.8 in parser v2.  We need to change it, so

Reply:
`getQueryString()` gets called when the RAML is 0.8, so we can't throw an exception. 
In 0.8 parser v2 the exception get thrown because it never gets called.